terra-jupyter-r/Dockerfile

```diff
@@ -131,6 +132,7 @@ RUN R -e 'install.packages("BiocManager")' \
     "nlme", \
     "nnet", \
     "rpart", \
+    "Seurat", \
```

Comment:
Not actually sure if `Seurat` should be in this image, or if we should leave it out (but allow it to be user-installable). Maybe will see how much it adds to the image

Reply:
It does increase the size but doesn't seem too bad:
```
seurat                                              latest    0afa8947e20f   2 hours ago     5.22GB
us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r      1.0.12    f472483aa606   2 weeks ago     4.64GB
```

Thinking of leaving it here -- it does seem to be a popular too for genomics.